components/filter_adult/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "MPL-2.0"
 
 [dependencies]
 base64 = "0.22.1"
-clap = { version = "4.5.36", default-features = false, features = ["std", "derive"], optional = true }
```

Comment:
This change doesn't hurt, but I don't think it's necessary either - we are already on a later semver compatible version and clap isn't touched in Cargo.lock here.

Reply:
Yeah, but `moz-central` is on `4.5.16` and our `Cargo.lock` doesn't affect things there.